reference_data/management/commands/update_gencode.py

```diff
@@ -25,7 +25,7 @@ class Command(BaseCommand):
 
     def add_arguments(self, parser):
         parser.add_argument('--reset', help="First drop any existing records from GeneInfo and TranscriptInfo", action="store_true")
-        parser.add_argument('--gencode-release', help="gencode release number (eg. 28)", type=int, required=True, choices=range(19, 32))
+        parser.add_argument('--gencode-release', help="gencode release number (eg. 28)", type=int, required=True, choices=range(19, 39))
```

Comment:
As the end of the range is exclusive, should this instead be this?

```suggestion
        parser.add_argument('--gencode-release', help="gencode release number (eg. 28)", type=int, required=True, choices=range(19, 40))
```

Do you also want to update the list at https://github.com/broadinstitute/seqr/blob/3d23adacf615d8a6423db28d44a087c6396a93dd/reference_data/management/commands/update_all_reference_data.py#L55-L59 accordingly?

Reply:
Ah, I'm a master at Python apparently :(
Yep, will action both.

Reply:
Feels like that should not update every single version of gencode though, should I actually just limit all of this to 39 totally? And then only update one gencode version?

Reply:
Do you mean whether to replace the list

```py
            update_gencode(31, reset=True)
            update_gencode(29)
            update_gencode(28)
            update_gencode(27)
            update_gencode(19)
```

with a simple

```py
            update_gencode(39, reset=True)
```

?

The comment in the code says:

> Old gene ids are used in the gene constraint table and other datasets, as well as older sequencing data

So maybe it's safer to just add 39 on top? Keeping the others shouldn't hurt, right?

Reply:
Then should I change the range to be `list(range(19, 32)) + [39]`, otherwise should I be adding update_genocode 31 through 39, or just 39.

Reply:
I don't think it matters too much. We could also just get rid of that range limit completely. I wouldn't worry about updating all intermediate versions, as we don't have data that would depend on those versions.

Reply:
I am happy either way - but if you are going to this effort now, should gencode 41 and bump to VEP 107 at the same time? We can hold out for vep108 if you would prefer, but I would prefer not to leave it past that.